deploy/farms.ts

```diff
@@ -0,0 +1,60 @@
+import { contract, createProxy, deploy, runIf } from 'ethereum-mars'
+import {
+  OwnedUpgradeabilityProxy,
+  LinearTrueDistributor,
+  TestTrustToken,
+  TimeOwnedUpgradeabilityProxy,
+  TrueMultiFarm,
+  TrustToken,
+} from '../build/artifacts'
+import { utils, BigNumber } from 'ethers'
+
+const DAY = 60 * 60 * 24
+
+const deployParams = {
+  mainnet: {
+    DISTRIBUTION_DURATION: 14 * DAY,
```

Comment:
Can we make the distribution duration for multifarm 2 years? We don't want to have to reset this every 2 weeks

Reply:
Sorry I missed this. I had just copied over the existing values from `deploy/truefi2.ts`, from the last time we ran this script for farms.

Reply:
Per offline discussion, also updating daily distribution to 330k TRU/day.